### PR TITLE
Updating C2S profile and CIS reference numbers with existing checks.

### PR DESCRIPTION
**Pull request by lfisher47**
While I was looking at the CIS vs C2S comparison, I decided to update some of the checks with the correct numbers for CIS version 2.1.1. I also added the check to C2S if the check existed. This is not a complete update, but I figured some was better than nothing.

### CODE REVIEW

**Comment by shawndwells** [comment]
Ok to test 

Shawn Wells

**Comment by shawndwells** [comment]
Thank you so much for this, @lfisher47! Looks like there are compile issues to clean up -- but this appears to be a great start.

Do you need help debugging the jenkins/build issues?

**Comment by lfisher47** [comment]
Yes please. I'm not understanding the error message and nothing looks out of place from my walk through of it, but I'm still learning how this all works.

File: shared/xccdf/services/imap.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@ POP3 server, the dovecot software should be disabled and removed.
 Running an IMAP or POP3 server provides a network-based
 avenue of attack, and should be disabled if not needed.
 </rationale>
-<ident prodtype="rhel7" cce="80294-2" />
+<ident prodtype="rhel7" cce="80294-2" cis="2.2.11" />
```
Comment (yuumasato) [inline shared/xccdf/services/imap.xml]: The log complains about ident without required attribute 'system', which is weird. The only change to `ident` is here.
```
File '/home/jenkins/workspace/scap-security-guide-pull-requests/label/ssg_new/build/RHEL/7/xccdf-unlinked-resolved.xml' line 31291: Element '{http://checklists.nist.gov/xccdf/1.1}ident': The attribute 'system' is required but missing.
OpenSCAP Error: Invalid XCCDF Checklist (1.1) content in /home/jenkins/workspace/scap-security-guide-pull-requests/label/ssg_new/build/RHEL/7/xccdf-unlinked-resolved.xml. [oscap_source.c:342]
```

Shouldn't`cis="2.2.11` go to `ref` element?

Comment (lfisher47) [reply shared/xccdf/services/imap.xml]: You are correct. Thanks for pointing that out. I'm still learning what all the error messages mean and how to find it.

**Comment by yuumasato** [comment]
Looks good, thank you for the updates. ACK.

**Pull request merged**